### PR TITLE
Correct wrong exception message when comparing different currencies 

### PR DESCRIPTION
**Pull request by benjaoming**
#225

Shuffling import because of isort

### CODE REVIEW

**Comment by benjaoming** [comment]
Maybe we should bump to 0.10 instead since it's breaking. That way, we can also introduce a fix for #220

**Comment by Stranger6667** [comment]
Hm, you're right. 0.10 is a way to go

**Comment by Stranger6667** [comment]
But I think, that it would be better to include some more changes to new release :) I'll submit some in a next few days

**Comment by benjaoming** [comment]
https://en.wikipedia.org/wiki/Release_early,_release_often :)

**Comment by benjaoming** [comment]
Don't stress adding more changes in order to release, it's totally okay to do a release with just one change IMO :)

**Comment by Stranger6667** [comment]
okey:)

File: CHANGES.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Changes in 0.9.2
+Changes in 0.10
 ----------------
 
 - Fixed ``understands_money`` behaviour. Now it can be used as a decorator `#215`_ (`Stranger6667`_)
+- Do not fail comparisons because of different currency. Just return ``False`` `#225`_ (`benjaoming` and `ivirabyan`)
```
Comment (Stranger6667) [inline CHANGES.rst]: To make links work properly it should be:
```
`benjaoming`_ and `ivirabyan`_
```

File: CHANGES.rst [hunk 2]
```diff
@@ -217,4 +218,4 @@ Changes in 0.3
 .. _toudi: https://github.com/toudi
 .. _tsouvarev: https://github.com/tsouvarev
 .. _w00kie: https://github.com/w00kie
-.. _willhcr: https://github.com/willhcr
\ No newline at end of file
+.. _willhcr: https://github.com/willhcr
```
Comment (Stranger6667) [inline CHANGES.rst]: Please, add

```
.. ivirabyan: https://github.com/ivirabyan
```

**Comment by benjaoming** [comment]
@Stranger6667 - thanks for the comment, addressed and pushed new changes + rebased to master.

**Comment by Stranger6667** [comment]
I'll fix sorting :) Thank you

**Pull request merged**